### ISSUE TITLE
Coordination

### ISSUE THREAD

**Issue by calebmrose00** [JoCaNo-Productions/flat-land#6]
Which champ should we work on next.
and also can we add the comment like

\# Status: Being Worked On (Caleb)
\# Status: Needs Review (Caleb)
ect.

to the end of functions and abilities to allow each other what we are working on because i was half way through making the ult for d.a.k. when i saw that you had submitted the full thing.

**Comment by calebmrose00** [comment]
I'm going to start working on Angelo, 
Xayah also looks easy to implement, after you finish on dak.

**Comment by calebmrose00** [comment]
Never mind i'll work on xayah.

**Comment by nmay231** [comment]
Yeah, I'll make sure to put the projects I am and will be working on in this issue.

To make commits easier, let's just choose a champ and finish him or her completely. If you or I can't think of a way to implement an ability, just put a comment in the champion discussion issue referencing the problem and a comment here.

You can also create the champion issue and assign yourself to it (top right on the issue page) to make it clear you're working on it. Then, when it's ready for review, assign me as well. We also need to make sure the wiki page gets updated.

**Comment by nmay231** [comment]
@calebmrose00 
* DaK is ready for review (as you saw--I'm sorry I wasted some of your time, ugh).
* I'll start reviewing Angelo
* I also claim to work on Stubby and Strawwoman, respectively

**Comment by nmay231** [comment]
* I'll flesh out Consumables/Items after you finish the `champ.end_of_turn()` feature. (Look at Gillan and champ.end_of_turn issues)

**Comment by nmay231** [comment]
* I reviewed Angelo, but the wording in the wiki stills needs to be solidified before he is finished.
Currently, I have the following todo items:
* Work on Stubby and Strawwoman, respectively
* Flesh out Items/Consumables
* I will review Xayah (since you will probably finish her first) and then Gillan.
* I'll go ahead and claim work on the `champ.end_of_turn()` thing. **DON'T START WORK ON IT PLEASE.**

**Comment by nmay231** [comment]
@calebmrose00 
With the comment: "`# Needs Review (Caleb)`", do you mean that _you_ need to review it or that it needs to be reviewed and you just added the comment? If the former, I didn't mean to barge into your work with Angelo. Apologies (and many a facepalm) if that is the case.

BTW, I will get a preliminary version of `Champion.end_of_turn()` pushed soon if not this afternoon.

**Comment by calebmrose00** [comment]
it mean that it needs to be reviewed and I added the comment.
and also you probably are the more equipped to review in the first place,
mr. Senior Software Engineer
